lib/cli/src/Cardano/CLI.hs

```diff
@@ -1084,11 +1084,9 @@ handleResponse encode res = do
             BL8.putStrLn (encode a)
         Left e -> do
             let msg = case e of
-                    FailureResponse r -> fromMaybe
+                    FailureResponse _ r -> fromMaybe
                         (T.decodeUtf8 $ BL.toStrict $ responseBody r)
                         (decodeError $ responseBody r)
-                    ConnectionError t ->
-                        t
```

Comment:
Before, the `ConnectionError` branch used to hold a `Text`, it now holds a `SomeException`, so the default case `T.pack $ show e` handles it just fine.